main.py

```diff
@@ -775,58 +778,58 @@ def parse_args(self) -> None:
         )
         parser.add_argument(
             "--commitHash",
-            help="Set the commit hash value",
+            help="The commit hash value from the SCM System",
             type=str,
             default=None,
             required=False,
         )
         parser.add_argument(
             "--branchName",
-            help="Set the branch name",
+            help="The name of the branch from the SCM System",
             type=str,
             default=None,
             nargs="*",
             required=False,
         )
         parser.add_argument(
             "--branchURI",
-            help="Set the branch URI",
+            help="The URI to the branch from the SCM System",
             default=None,
             required=False,
         )
         parser.add_argument(
             "--buildVersion",
-            help="Set the build version",
+            help="Version of application build artifacts",
             type=str,
             default=None,
             required=False,
         )
         parser.add_argument(
             "--buildURI",
-            help="Set the build URI",
+            help="URI to CI build info",
             type=str,
             default=None,
             required=False,
         )
         parser.add_argument(
             "--operatingEnvironment",
-            help="Set the Operating Environment",
+            help="Operating environment for information porpuses only",
```

Comment:
does this return the operating environment, or set it?

Reply:
changed the wording to be more clear.